04-request-lifecycle/01-nestjs-components/filters/http-error.filter.ts

```diff
@@ -1,5 +1,25 @@
-import { ArgumentsHost, ExceptionFilter } from "@nestjs/common";
+import { ArgumentsHost, Catch, ExceptionFilter, HttpException } from "@nestjs/common";
+import { Request, Response } from 'express';
+import {appendFileSync} from 'fs';
 
+@Catch(HttpException)
```

Comment:
Вот здесь мы на самом деле хотим перехватывать все ошибки (для тренировки в первую очередь), в тестах выбрасывается обычная Error, поэтому в наш фильтр она не попадает. Нужно изменить на 
```suggestion
@Catch()
```

Reply:
Спасибо, поправил